portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -10,15 +10,17 @@
     private final Date date;
     private final String emotion;
     private final boolean isAbleToDelete;
+    private final String id;
```

Comment:
Any reason you're choosing to store the UUID as a string vs a UUID object?

Reply:
Ids in Datastore are kept as String and I am using Comment objects to create JSON objects. I think keeping ids as String makes most sense, since otherwise I would need to convert UUID into String and the other way around in several places.